crates/floresta-wire/src/p2p_wire/address_man.rs

```diff
@@ -389,6 +389,16 @@ impl AddressMan {
         Ok(())
     }
 
+    pub fn dump_utreexo_peers(&self, datadir: &str) -> std::io::Result<()> {
+        let peers_id = self.good_peers_by_service.get(&service_flags::UTREEXO.into()).unwrap();
+        let addresses: Vec<_> = peers_id.iter().map(|id| self.addresses.get(id).unwrap().to_owned()).map(Into::<DiskLocalAddress>::into).collect();
+        let addresses = serde_json::to_string(&addresses);
+        if let Ok(addresses) = addresses {
+            std::fs::write(datadir.to_owned() + "/anchors.json", addresses)?;
+        }
+        Ok(())
+    }
+
```

Comment:
Please, add docs for new functions.

Reply:
I have added simple docs to keep it clear and concise. If needed I can expand them.